src/picongpu/include/simulation_defines/unitless/physicalConstants.unitless

```diff
@@ -65,9 +65,9 @@ namespace picongpu
 
 
     /** Unit of mass */
-    BOOST_CONSTEXPR_OR_CONST float_64 UNIT_MASS = SI::ELECTRON_MASS_SI * double(particles::TYPICAL_NUM_PARTICLES_PER_MACROPARTICLE);
+    BOOST_CONSTEXPR_OR_CONST float_64 UNIT_MASS = SI::BASE_MASS_SI * double(particles::TYPICAL_NUM_PARTICLES_PER_MACROPARTICLE);
     /** Unit of charge */
-    BOOST_CONSTEXPR_OR_CONST float_64 UNIT_CHARGE = -1.0 * SI::ELECTRON_CHARGE_SI * double(particles::TYPICAL_NUM_PARTICLES_PER_MACROPARTICLE);
```

Comment:
@axel: I think if we derive from a base value than we need to skip the `-1.0` in front. This is only in because of the fact that we typically used electrons as base.